src/App.tsx

```diff
@@ -15,71 +16,101 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+function getReorderedGoods(
+  goods: string[],
+  sortType: SortType,
+  isReversed: boolean,
+) {
+  const visibleGoods = [...goods];
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  visibleGoods.sort((g1, g2) => {
+    switch (sortType) {
```

Comment:
switch/case should be used with `default` statements. You can return some default value (for sort for example `0`)